docs/aws/README.md

```diff
@@ -1,11 +1,31 @@
-# AWS
+# Getting Started with Karpenter on AWS
 
-This guide will provide a complete Karpenter installation for AWS.
-These steps are opinionated and may need to be adapted for your use case.
+Karpenter automatically provisions new nodes in response to unschedulable  pods. Karpenter does this by observing events within the kubernetes cluster, and then sending commands to the underlying cloud provider. 
 
-> This guide should take less than 1 hour to complete and cost less than $.25
+In this example, the cluster is running on Amazon Web Services (AWS) Elastic Kubernetes Service (EKS). Karpenter is designed to be cloud provider agnostic, but currently only supports AWS. Contributions are welcomed. 
+
+This guide should take less than 1 hour to complete, and cost less than $0.25. Follow the clean-up instructions to reduce any charges.
+
+# Install
+
+Karpenter is installed in clusters with a simple helm chart.
+
+Karpenter additionally requires IAM Roles for Service Accounts (IRSA). IRSA permits Karpenter (within the cluster) to make privileged requests to AWS (as the cloud provider). 
+
+## Required Utilities
+
+Install these tools before proceeding:
+
+1. [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-linux.html)
+2. `kubectl` - [the kubernetes CLI](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/)
```

Comment:
Any reason why we don't just say "[`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/)"

Reply:
links inside code fmt can render inconsistently,

Reply:
Sure, but can we at least make it consistent with the other list items.